content/en/docs/installation/upgrading/remove-deprecated-apis.md

```diff
@@ -14,27 +14,110 @@ We have deprecated the following cert-manager APIs:
 - `acme.cert-manager.io/v1alpha3`
 - `acme.cert-manager.io/v1beta1`
 
-These APIs were removed in cert-manager `v1.6.0`. If you have a cert-manager installation that is using or has previously used these deprecated APIs you'll need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6.0`.
+These APIs are no longer served in cert-manager `v1.6` and are fully removed in cert-manager `v1.7`. If you have a cert-manager installation that is using or has previously used these deprecated APIs you'll need to upgrade your cert-manager custom resources and CRDs. This should be done before upgrading to cert-manager `v1.6`.
 
-## Upgrading existing cert-manager resources
+{{% alert title="Warning" color="warning" %}}
+Although the deprecated APIs are no longer served in cert-manager `v1.6`, it is possible that at this point some cert-manager resources are still stored in `etcd` at the deprecated API versions and that the deprecated versions are still recorded on the cert-manager `CustomResourceDefinition`s' statuses.
+You can verify the storage versions on CRD status fields by running `kubectl get crd <crd-name>  -ojsonpath='{.status.storedVersions}`, but we are not aware of any straightforward way how to verify what versions of resources are stored in `etcd`.
```

Comment:
This sentence is a little confusing IMO - what does it mean/what's the reason for verifying the storage version on CRD status fields if this *doesn't* reflect what's in etcd? What's actionable here from an end-user standpoint? :)

Reply:
The upgrade will fail if there is a removed storage version recorded on CRD fields no matter whether the resources have been stored in etcd at v1 or not

Reply:
👍 yep that makes sense - but I don't think this message is making that clear. Can we adjust it to be a bit more actionable for a user?

"Before the upgrade can proceed, you must ensure that the `status.storedVersions` field only contains `v1`. If the upgrade is attempted whilst there are still other versions recorded in this field, the upgrade will fail and your installation may be left inoperable until it is resolved." or something like that?

Reply:
I _think_ I've come across some upstream issues where the point of the stored versions is explained, but cannot find those now..

Reply:
> yep that makes sense - but I don't think this message is making that clear. Can we adjust it to be a bit more actionable for a user?

That warning was there because we previously believed (and told users) that upgrade to v1.6 would show if there are resources at the deprecated versions in the cluster (as v1.6 stops no longer serves those APIs)- it is meant to convey that it's possible to be at v1.6 and not notice that you have old resources/not updated CRD status fields. It's not meant to be a general warning that CRD statuses need to be updated as that is already what this doc is about.
Agree that it's not best worded, I can remove it 👍🏼

Reply:
I've changed the contents to refer to the cmctl install command